lua/lazyvim/plugins/extras/lang/zig.lua

```diff
@@ -2,8 +2,8 @@ return {
   {
     "NTBBloodbath/zig-tools.nvim",
     ft = "zig",
-    config = function()
-      require("zig-tools").setup()
+    opts = function(_, opts)
+      require("zig-tools").setup(opts)
```

Comment:
Why are you trying to `require` the module? That's already what `lazy.nvim` does. A simple `opts = {}` should do it I believe (since setting `opts` implies the call to `Plugin.config` which calls the setup function of the module). In my opinion, this is redundant.

Reply:
No need to worry. I wasn't criticizing you or anything. Just pointed out how it could be simplified, since you weren't customizing the `config` function yourself and the plugin doesn't need some specific need of setup, like for example `nvim-treesitter` which needs to be setup by doing `require'nvim-treesitter.configs'.setup` and `lazy.nvim` wouldn't be able to call the correct module on its own based on the heuristics it uses to identify the correct `main` module. Keep up the good work. I'm just a newbie like you. `lazy.nvim`'s docs are a treasure providing great knowledge and I myself have to revisit them from time to time to clarify things, so I suggest you give them a good read to understand better how things work.